cookbooks/fb_networkmanager/recipes/default.rb

```diff
@@ -0,0 +1,100 @@
+#
+# Cookbook:: fb_networkmanager
+# Recipe:: default
+#
+# Copyright (c) 2019-present, Vicarious, Inc.
+# Copyright (c) 2019-present, Facebook, Inc.
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+vpnhome = '/var/lib/openvpn/chroot'
+FB::Users.initialize_group(node, 'nm-openvpn')
+node.default['fb_users']['users']['nm-openvpn'] = {
```

Comment:
should we add `fb_users` as a depends in metadata?

Reply:
Still relevant

Reply:
Oh - sorry. Yes, in theory. In reality you haven't merged that yet, so what I should *probably* do is something like:
```
if node['fb_users']
  ...
end
```
so people who are using it get the benefit and people who aren't don't need to.

Reply:
I did the above `if`